src/extern/bpf/bpf_helpers.h

```diff
@@ -0,0 +1,58 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
```

Comment:
there is a bpf_helpers.h under src/extern/. It seems content different than the one in this PR.

Reply:
This helper class is for bpf.h